### ISSUE TITLE
Update locks up Keepass2android when data connection is bad

### ISSUE THREAD

**Issue by wellread1** [PhilippC/keepass2android#1677]
I regularly have need of Keepass2android when I am in area with poor mobile reception.  However, I find that when I open Keepass2android at this location and an update is pending the app locks up.  I can't recover by rebooting or putting the phone into airplane mode after Keepass2android is locked-up.  I have to move to a location where I have a good data connection, which is not practical in the particular case.

Most recently this happened May 16, 2021 (today).  When I finally got to an area of good reception the "Donate a beer" splash screen displayed, Keepass2android began working, and when I checked my version it was 1.08d-r5 (Mar 1, 2021).  However, Google play store showed a 24MB pending update.  When I pressed update I received the message "Auto update disabled" which was not unexpected because I had previously disabled auto updates to try to prevent the aforementioned problem. After the update the installed version is 1.09a-pre1.  Curiously the change log noted that I had been reverted to a previous version.  Note:  Some months ago I left the beta program to avoid extra updates.

Bottom line is I keep getting locked out of Keepass2android at this location unless I remember to make sure that KeePass2android is up-to-date and place the phone in airplane mode before I go to the location.

I don't know if there is anything you can do about the update procedure but it can't hurt to bring this to your attention.
